src/Chainweb/RestAPI/NodeInfo.hs

```diff
@@ -56,15 +60,21 @@ data NodeInfo = NodeInfo
 instance ToJSON NodeInfo
 instance FromJSON NodeInfo
 
-nodeInfoHandler :: ChainwebVersion -> Server NodeInfoApi
-nodeInfoHandler v = return
-  NodeInfo
-  { nodeVersion = v
-  , nodeApiVersion = prettyApiVersion
-  , nodeChains = (chainIdToText <$> (toList $ chainIds v))
```

Comment:
@larskuhtz Do we need to add a `BlockHeight` parameter to this `chainIds` function?